docs/basic-features/environment-variables.md

```diff
@@ -47,6 +47,7 @@ npx cross-env NEXT_PUBLIC_EXAMPLE_KEY=my-value next dev
 
 - Trying to destructure `process.env` variables won't work due to the limitations of webpack's [DefinePlugin](https://webpack.js.org/plugins/define-plugin/).
 - To avoid exposing secrets, do not use the `NEXT_PUBLIC_` prefix for them. Instead, [expose the variables using `.env`](#exposing-environment-variables).
+- If you are migrating from the `dotenv` npm package, remove this package first, otherwise you'll be debugging why your variables are showing up as `undefined`.
```

Comment:
```suggestion
- You cannot have `dotenv` installed in your project, as this will cause Next.js to disable the auto-loading of the environment variables. Look for and uninstall this package if your variables are showing us as `undefined`.
```

Reply:
Thanks @Timer!